functions/lambda_function.py

```diff
@@ -0,0 +1,17 @@
+difference = lambda c, d : c - d
```

Comment:
stop using a, b, c, d, x, y... for variable names always give meaningful names
```suggestion
difference = lambda operand1, operand2 : operand1 - operand2
```
```suggestion
difference = lambda c, d : c - d
```